.github/mingw-build.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+# set -e
+
+cd "$(dirname "$0")"
+
+buildCyanrip() {
+    cyan_prepare
+    build_curl
+    build_neon
+    build_discid
+    build_libmusicbrainz
+    build_ffmpeg
+    build_cyanrip
+}
+
+cyan_prepare() {
+    local rootdir="$(pwd)"
+    CYANRIPBUILDDIR="$(realpath "$rootdir/_build")"
+    CYANRIPINSTALLDIR="$(realpath "$rootdir/_install")"
+    PKG_CONFIG_PATH=$CYANRIPINSTALLDIR/lib/pkgconfig:$PKG_CONFIG_PATH
+    CPATH="`cygpath -m $CYANRIPINSTALLDIR/include`;`cygpath -m $MINGW_PREFIX/include`"
```

Comment:
```suggestion
    CPATH="$(cygpath -pm "$CYANRIPINSTALLDIR/include:$MINGW_PREFIX/include")"
```

you can use the `p` to make it autoconcatinate to a path

```none
  -p, --path            NAME is a PATH list (i.e., '/bin:/usr/bin')
```